src/config/dynamic-theme-fixes.config

```diff
@@ -2319,6 +2319,20 @@ CSS
 
 ================================
 
+bloomberg.com
+
+INVERT
+.navi-bar__button-icon
+.navi-sections__item--LiveNow::before
+.navi-sections__link--current::after
+
+CSS
+.navi {
+    border-color: rgb(140, 130, 115) !important;
+}
```

Comment:
Could this use the `${original color}` for the color part?

Reply:
Good idea. It was a solid black border turned off-white, but using a var() it looks good in alternative color schemes.